### ISSUE TITLE
Documentation on A11y

### ISSUE THREAD

**Issue by EvanBacon** [expo/expo#6558]
Need a doc cover how to use A11Y universally in Expo
- How to debug A11Y for iOS, Android, Web
- Best practices 
- Using System font sizes
- Dark mode
- Localization

**Comment by mtimofiiv** [comment]
This is a super important issue. It doesn't seem right to turn off font scaling, for example, using `allowFontScaling: false` on `Text` nodes, but an option that respects the user's settings would certainly be nice.

**Comment by dr-thca** [comment]
From what I've seen a11y is still severally lacking in RN; like not being able to set the focus order: (https://appt.org/en/docs/react-native/samples/accessibility-order and https://github.com/react-native-community/discussions-and-proposals/discussions/389). Finding this crucial info is sort of spread out, so having some documentation and pointers would really help I think. (Or maybe even releasing a library for it? https://www.npmjs.com/package/react-native-a11y seems like the only option right now)
